sdk/storage/azure-storage-file/tests/test_get_file.py

```diff
@@ -789,16 +789,17 @@ def test_ranged_get_file_to_path_invalid_range_parallel(self):
         file_client.upload_file(file_data)
 
         # Act
+        start = 3
```

Comment:
Why is this set to 3? The tests below seem to use the offset value of 1

Reply:
Do these values work? They don't align with 512 - but maybe it doesn't matter for these tests?

Reply:
Yes...download_file doesn't need aligning with 512.